season-11/machines.json

```diff
@@ -0,0 +1 @@
,"Aerosmith":{"key":"Aerosmith","name":"Aerosmith"},"NBA":{"key":"NBA","name":"Stern NBA"},"Jokerz":{"key":"Jokerz","name":"Jokerz"},"Buckaneer":{"key":"Buckaneer","name":"Buccaneer"},"Catacomb":{"key":"Catacomb","name":"Catacomb"},"PistolPoker":{"key":"PistolPoker","name":"Pistol Poker"},"GameShow":{"key":"GameShow","name":"Game Show"},"Blackjack":{"key":"Blackjack","name":"Blackjack"},"Viper":{"key":"Viper","name":"Viper Night Driving"},"Stargate":{"key":"Stargate","name":"Stargate"},"SternWars":{"key":"SternWars","name":"Star Wars (Stern)"},"DND":{"key":"DND","name":"Dungeons and Dragons"},"EBC":{"key":"EBC","name":"Eight Ball Champ"},"Future Spa":{"key":"Future Spa","name":"FutureSpa"},"Hard Body":{"key":"Hard Body","name":"Hardbody"},"Hotdoggin'":{"key":"Hotdoggin'","name":"Hotdoggin"},"DialedIn":{"key":"DialedIn","name":"Dialed In!"},"SM":{"key":"SM","name":"Silverball Mania"},"TruckStop":{"key":"TruckStop","name":"Truck Stop"},"Playboy":{"key":"Playboy","name":"Playboy (Bally)"},"Shrek":{"key":"Shrek","name":"Shrek"},"Frankenstein":{"key":"Frankenstein","name":"Frankenstein"},"Jetsons":{"key":"Jetsons","name":"Jetsons"},"Torpedo":{"key":"Torpedo","name":"Torpedo Alley"},"NR":{"key":"NR","name":"Night Rider"},"SixMil":{"key":"SixMil","name":"Six Million Dollar Man"},"KoS":{"key":"KoS","name":"Kings Of Steel"},"Baywatch":{"key":"Baywatch","name":"Baywatch"},"Skateball":{"key":"Skateball","name":"Skateball"},"MarsTrek":{"key":"MarsTrek","name":"Mars Trek"},"Gaurdians":{"key":"Gaurdians","name":"Guardians of the Galaxy"},"TNA":{"key":"TNA","name":"Total Nuclear Annihilation"},"PlayboyDE":{"key":"PlayboyDE","name":"Playboy (Data East)"},"PlayboyStern":{"key":"PlayboyStern","name":"Playboy (Stern)"},"Guardians":{"key":"Guardians","name":"Guardians of the Galaxy"},"StarRace":{"key":"StarRace","name":"Star Race"},"TagTeam":{"key":"TagTeam","name":"Tag Team"},"SpaceInvaders":{"key":"SpaceInvaders","name":"Space Invaders"},"BMX":{"key":"BMX","name":"BMX"},"BeamOn":{"key":"BeamOn","name":"Beam On"},"LW3":{"key":"LW3","name":"Lethal Weapon 3"},"Houdini":{"key":"Houdini","name":"Houdini"},"TBL":{"key":"TBL","name":"The Big Lewbowski"},"Arena":{"key":"Arena","name":"Arena"},"SpyHunter":{"key":"SpyHunter","name":"Spy Hunter"},"Apollo13":{"key":"Apollo13","name":"Apollo 13"},"DP":{"key":"DP","name":"Dead Pool"},"IronMaiden":{"key":"IronMaiden","name":"Iron Maiden"},"Sharpshooter":{"key":"Sharpshooter","name":"Sharpshooter"},"StrangeScience":{"key":"StrangeScience","name":"Strange Science"},"Transporter":{"key":"Transporter","name":"Transporter The Rescue"},"MarioMushroom":{"key":"MarioMushroom","name":"Super Mario Mushroom World"},"Force2":{"key":"Force2","name":"Force II"},"LW":{"key":"LW","name":"Lost World Bally"},"PinballMagic":{"key":"PinballMagic","name":"Pinball Magic"},"WildFyre":{"key":"WildFyre","name":"Wild Fyre"},"FlashGordon":{"key":"FlashGordon","name":"Flash Gordon"},"POTCJJ":{"key":"POTCJJ","name":"Pirates of the Caribbean (Jersey Jack)"}}
```

Comment:
Hmm...it's probably worth looking at removing the `key` field, since it is the same as the actual key in the object. Need to track down the spot in the code.